Kitodo-API/src/main/java/org/kitodo/api/dataformat/service/DivServiceInterface.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataformat.service;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Interface for a service that handles access to the {@code <mets:div>}
+ * element.
+ *
+ * <p>
+ * A div is a node of an outline tree, which divides the description of the
+ * digitized work according to the principle of a table of contents. Each div
+ * refers to an ordered list of one or more views (METS idiom: areas) that mimic
+ * the section of the digitized work described by this node when consumed in the
+ * given order. Each div can be assigned an amount of meta-data describing the
+ * described section of the digitized work in abstracted form.
+ */
+public interface DivServiceInterface {
+    /**
+     * Returns services to access the views (METS idiom: areas) that mimic the
+     * section of the digitized work associated with this div. Since this area
+     * is currently not fully developed, it can currently be assumed that the
+     * number of areas is always 1.
+     *
+     * @return the areas of this div
+     */
+    List<AreaServiceInterface> getAreas();
+
+    /**
+     * Returns services for access to the outline level that is a child of this
+     * outline level.
+     *
+     * @return services for access to the children of this div
+     */
+    List<DivServiceInterface> getChildren();
+
+    /**
+     * Returns the human-readable name of this outline element.
+     *
+     * @return the label
+     */
+    String getLabel();
+
+    /**
+     * Returns all meta-data that describe this outline level.
+     *
+     * @return the meta-data
+     */
+    Collection<MetaDataServiceInterface> getMetaData();
```

Comment:
MetaData or MetaDataService? I think metadata is an one word: https://www.wikiwand.com/en/Metadata

Reply:
After all that I know, the only real corret spelling is “meta-data”, with dash. Therefore the camel case.

See [this excellent blog post](http://tdan.com/the-meta-data-fiasco/18502), starting from about one screen down where it says: “A second misperception with meta-data is the spelling.”

Reply:
I agree with @Beacze that we should adopt the spelling as `metadata`. We can't go by a blogpost or the whole history of the word but how it is used today. (I know that might be hard for you @matthias-ronge since your background is in linguistics.) Even the blog posts says `Meta must be combined with a root word, either with a hyphen or by concatenation` So that's what we're gonna do.

 Every dictionary I checked uses the single word spelling and therefore we use it that way.

Reply:
No, it's not that bad for me. I just wanted to point out that there is another view here as well. And I was once told by an American that Production would be so difficult to understand because the English would be used so wrong in part. That's why I try a little bit to pay attention. But now it's clearly 2 to 1, so I've changed it now.